File: app/src/main/java/com/main/climbingdiary/database/entities/Area.java
package com.main.climbingdiary.database.entities;

import org.chalup.microorm.annotations.Column;

import lombok.Data;
import lombok.ToString;

@ToString
@Data
public class Area {
    @Column("land") private String country;
    @Column("name") private String name;
    @Column("id") private int id;
    @Column("lat") private double lat;
    @Column("lng") private double lng;
}

File: app/src/main/java/com/main/climbingdiary/database/entities/Area.kt
package com.main.climbingdiary.database.entities

import org.chalup.microorm.annotations.Column

data class Area(
    @Column("land") var country: String = "",
    @Column("name") var name: String,
    @Column("id") var id: Int = 0,
    @Column("lat") var lat: Double = 0.0,
    @Column("lng") var lng: Double = 0.0
)